### PR TITLE
subiendo seccion de ventas flash

### PR DESCRIPTION
**Pull request by Sgutierrezgf**
Agregando seccion de ventas rapidas

### CODE REVIEW

File: apps/web/src/components/Home/FlashSale/index.tsx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+                        <div className='w-full overflow-hidden p-2 ps-0'>
+                            <h2 className='truncate mb-1 font-semibold md:mb-1.5 text-sm sm:text-base md:text-sm lg:text-base xl:text-lg text-heading'>
+                                Zara Shoes Green
+                            </h2>
+                            <p className='text-body text-xs lg:text-sm leading-normal xl:leading-relaxed max-w-[250px] truncate'>
+                                Footwear refers to garments worn
+                            </p>
+                            <div className='font-semibold text-sm sm:text-base mt-1.5 space-s-2 sm:text-xl md:text-base lg:text-xl md:mt-2.5 2xl:mt-3 text-heading'>
+                                <span className='inline-block false'>$50.00</span>
+                                {/* <del className='sm:text-base font-normal text-gray-800 pl-3'>$16.00</del> */}
```
Comment (Ragnarok99) [inline apps/web/src/components/Home/FlashSale/index.tsx]: could we please delete this if it's not being used?

Comment (Sgutierrezgf) [reply apps/web/src/components/Home/FlashSale/index.tsx]: ready

File: apps/web/src/components/Home/FlashSale/index.tsx [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+                            <p className='text-body text-xs lg:text-sm leading-normal xl:leading-relaxed max-w-[250px] truncate'>
+                                Monochrome elegance.
+                            </p>
+                            <div className='font-semibold text-sm sm:text-base mt-1.5 space-s-2 sm:text-xl md:text-base lg:text-xl md:mt-2.5 2xl:mt-3 text-heading'>
+                                <span className='inline-block false'>$12.00</span>
+                                <del className='sm:text-base font-normal text-gray-800 pl-3'>$16.00</del>
+                            </div>
+                        </div>
+                    </div>
+                </div>
```
Comment (Ragnarok99) [inline apps/web/src/components/Home/FlashSale/index.tsx]: I think we can wrap this into a reusable component, we could just pass it props for the `name, image, price, description, etc` what do you think?

Comment (Sgutierrezgf) [reply apps/web/src/components/Home/FlashSale/index.tsx]: yeah, i will see i can do

**Comment by Ragnarok99** [comment]
this is how it looks on mobile right now 
Screen Shot 2022-11-15 at 3 14 35 PM

this can be improved in a later PR or If you want to go one step further in this PR and add that feel free to do it 🚀

**Review by Ragnarok99 — APPROVED** [review]
nice job!! 🚀

**Pull request merged**